mobly/controllers/android_device_lib/adb.py

```diff
@@ -134,6 +136,21 @@ def list_occupied_adb_ports():
   return used_ports
 
 
+def can_adb_pick_available_forwarding_port():
+  """True if the ADB can pick an available forwarding port; False otherwise.
+
+  By default, the ADB binary supports this feature, i.e. it supports passing 0
+  as host port when doing port forwarding.
+
+  But sometimes users override the `ADB` variable to use a customized variant to
```

Comment:
can we afford to wait for the migration to adb forwarder to push this PR, or is there some urgency?

Reply:
This is not urgent, we can wait for the migration.